docs/core/testing/unit-testing-platform-intro.md

```diff
@@ -209,6 +209,17 @@ Specifies the minimum number of tests that are expected to run. By default, at l
 
 The directory where the test results are going to be placed. If the specified directory doesn't exist, it's created. The default is `TestResults` in the directory that contains the test application.
 
+## MSBuild integration
+
+The NuGet package [Microsoft.Testing.Platform.MSBuild](https://www.nuget.org/packages/Microsoft.Testing.Platform.MSBuild) provides various integrations for `Microsoft.Testing.Platform` with MSBuild.
+
+- Support for `dotnet test`, see [dotnet test integration](./unit-testing-platform-integration-dotnet-test.md) for more information.
+- Support for `ProjectCapability` required by `Visual Studio` and `Visual Studio Code` Test Explorers.
+- Automatic generation of the entry point (`Main` method)
```

Comment:
We're also generating the configuration file.

Reply:
```suggestion
- Automatic generation of the entry point (`Main` method)
- Automatic generation of the configuration file
```